docs/databases/hadoop/install-and-configure-hadoop-cluster.md

```diff
@@ -0,0 +1,519 @@
+
+There are lot more commands to manage your HDFS, you can look at the [Apache HDFS shell documentation](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/FileSystemShell.html), or print help with:
+
+    hdfs dfs -help
+
+## Running YARN
+
+HDFS is only a distributed storage system, it does not provide any services for running and scheduling tasks in the cluster. This is the role of the YARN framework. This section is about starting, monitoring and submitting jobs to YARN.
+
+### Starting and Stopping YARN
+
+1. Start YARN with the script:
+
+        start-yarn.sh
+
+2. Check that everything is running with `jps` command. You should see in addition to previous HDFS daemon, a **ResourceManager** on node-master, and a **NodeManager** on node1 and node2.
+
+3. To stop YARN, run the following command on node-master:
+
+        stop-yarn.sh
+
+### Monitoring YARN
+
+1. The `yarn` command provides utilities to manage your YARN cluster, like publishing and running applications. You can also get a report of running nodes with the command:
+
+        yarn node -list
+
+    Similarly, you can get a list of running applications with command:
+
+        yarn application -list
+
+    To get all available parameters of the `yarn` command, see [Apache YARN documentation](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YarnCommands.html).
+
+2. As with HDFS, YARN provides a more friendly web UI, started by default on port 8088 of the ResourceManager. Point your browser to http://node-master:8088 and browse the UI:
+
+![Screenshot of YARN Web UI](/docs/assets/hadoop/hadoop-4-yarn-webui-wide.png)
+
+### Submitting MapReduce Jobs to YARN
+
+Yarn jobs are packaged into jar files and submitted to YARN for execution with the command `yarn jar`. Hadoop installation package provides sample applications that can be run to test your cluster. We will use them to run a word count on the three books we previoulsy upload to HDFS.
+
+1. Submit a job with the sample jar to YARN. On node-master, run:
+
+        yarn jar ~/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.8.1.jar wordcount "books/*" output
+
+    The last argument is where output of the job will be saved, in HDFS.
+
+2. After the job is finished, you can get the result by querying HDFS with `hdfs dfs -ls output`. In case of success, the output will resemble:
+
+        Found 2 items
+        -rw-r--r--   1 hadoop supergroup          0 2017-10-11 14:09 output/_SUCCESS
+        -rw-r--r--   1 hadoop supergroup     269158 2017-10-11 14:09 output/part-r-00000
+
+3. Print the result with:
+
+        hdfs dfs -cat output/part-r-00000
+
+
+## What to Do Next
+
+Now that you have a up and running YARN cluster, you can:
+
+-  Learn how to code your own YARN jobs with [Apache documentation](hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/WritingYarnApplications.html)
+- Install Spark on top on your YARN cluster with [Linode Spark guide](to/be/confirmed)
```

Comment:
Before publishing this last line, the path should be updated to the coming Spark guide.